### PR TITLE
LPS-147393 Add Fragments Advanced panel

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/selectors/selectPanels.js [hunk 1]
```diff
@@ -190,14 +197,24 @@ export function selectPanels(activeItemId, activeItemType, state) {
 		const fieldSets = fragmentEntryLink?.configuration?.fieldSets ?? [];
 
 		panelsIds = {
+			[PANEL_IDS.fragmentAdvanced]:
+				canUpdateItemConfiguration &&
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/selectors/selectPanels.js:201]: canUpdateItemConfiguration is is all types but editables, can we simplify this a bit?, probably an early return

**Comment by victorg1991** [comment]
LGTM :) Just a couple of suggestions

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/FragmentAdvancedPanel.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+		},
+		[dispatch, fragmentEntryLink, languageId]
+	);
+
+	const fieldSets = fragmentEntryLink.configuration?.fieldSets?.filter(
+		(fieldSet) =>
+			fieldSet.configurationRole === FRAGMENT_CONFIGURATION_ROLES.advanced
+	);
+
+	if (!fieldSets.length) {
```
[the remark this answers is not in the document]
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/FragmentAdvancedPanel.js:59]: That's right, thanks 😛

**Comment by sandrodw3** [comment]
ci:forward

**Pull request closed without merging**